app/views/pages/homepage.html.erb

```diff
@@ -71,7 +71,7 @@
         <ul>
           <li><%= link_to raw("Sign In to #{abbr_tag('dps')}"), 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %>  to schedule your move online</li>
           <li>Visit your <%= link_to 'local personal property office', offices_path %> to get help scheduling your move</li>
-          <li><%= link_to 'Follow online tutorials for help logging in or using DPS', tutorials_path %></li>
+          <li><%= link_to 'Follow online tutorials for help logging in or using DPS', page_path('tutorials') %></li>
```

Comment:
I _think_ you could use `tutorial_path` without additional params here and elsewhere to link to `/tutorials`.

Reply:
…though that's kinda awkward, too. 🤔

Reply:
Yep I could but it felt semantically wrong